wrappers/android/examples/capture/src/main/java/com/intel/realsense/capture/MainActivity.java

```diff
@@ -154,7 +155,7 @@ private void configAndStart() throws Exception {
         {
             config.enableStream(StreamType.DEPTH, 640, 480);
             config.enableStream(StreamType.COLOR, 640, 480);
-            mPipeline.start(config);
+            try(PipelineProfile pp = mPipeline.start(config)){}
```

Comment:
If the pattern must be applied throughout the user code, then the rationale must be documented as well.

Reply:
Where can I add documentation for it?

Reply:
Comments added in code